### PR TITLE
Remove success message when toggling preview-edit modes.

### PR DESCRIPTION
**Pull request by vkaracic**
KF-718

### CODE REVIEW

File: views/lti/editorMethods.jade [hunk 1]
```diff
@@ -75,19 +75,23 @@ script
...
+            const form = event.editor.element.$.parentNode;
+            saveData(form, event.editor.getData());
+          }, 5000)
+        });
       }
 
       function saveData(form, data=undefined) {
-        const successMessage = form.querySelector('.success-message')
-        const globalSuccessMessage = document.querySelector('.canvas-control-container .success-message')
+        const canvasForm = form.classList.contains('canvas-form');
```
Comment (mbareta) [inline views/lti/editorMethods.jade]: Rename to isCanvasForm to note that the var is a boolean.

File: views/lti/editorMethods.jade [hunk 2]
```diff
@@ -75,19 +75,23 @@ script
...
+        const successMessage = (canvasForm)
```
Comment (mbareta) [inline views/lti/editorMethods.jade]: You can remove brackets here.

File: views/lti/editorMethods.jade [hunk 3]
```diff
@@ -28,7 +28,7 @@ script
   :babel
     function initializeCKEditor(isBmc = false) {
         CKEDITOR.on('instanceCreated', function (e) {
-          e.editor.on('change', autosave);
+          e.editor.on('focus', autosave);
```
Comment (mbareta) [inline views/lti/editorMethods.jade]: Make sure to unbind the event on blur or we will have a new event listener registered every time a user clicks on the editor.

**Comment by vkaracic** [comment]
@mbareta take another look, but please don't merge.

**Comment by mbareta** [comment]
:+1:

**Pull request merged**